cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -264,3 +265,21 @@ def test_reports_delete_saved_report(appliance, request):
     view.configuration.item_select(
         item='Delete selected Saved Reports', handle_alert=True)
     assert not report.exists
+
+
+def test_crud_custom_report_schedule(
+    appliance, request, custom_report_values, schedule_data
+):
+    """This test case creates a schedule for custom reports and tests if it was created
+    successfully.
+    """
+    custom_report = appliance.collections.reports.create(**custom_report_values)
+    schedule_data["filter"] = (
+        "My Company (All Groups)",
+        "Custom",
+        custom_report.menu_name,
+    )
+    custom_report_schedule = appliance.collections.schedules.create(**schedule_data)
+    assert custom_report_schedule.exists
+    custom_report_schedule.delete(cancel=False)
+    request.addfinalizer(custom_report.delete)
```

Comment:
Required:
Move this `Finalizer` after `L276` it will increase readability. I think, creating `custom_report` and deletion is a part of a `setup` for this test. If possible, I would suggest you to create separate `fixture` which will create `custom_report` for you and delete at the time of `tear down`.

Reply:
@digitronik I've added a separate fixture `get_custom_report` for the test.